src/main/java/net/minecraftforge/common/Tags.java

```diff
@@ -500,10 +584,72 @@ private static void init() {}
          * Holds all fluids that are gaseous at room temperature.
          */
         public static final TagKey<Fluid> GASEOUS = tag("gaseous");
+        /**
+         * Holds all fluids related to honey. (Standard unit for honey bottle is 250mb per bottle)
+         */
+        public static final TagKey<Fluid> HONEY = tag("honey");
+        /**
+         * Holds all fluids related to potions. The effects of the potion fluid should be read from NBT.
```

Comment:
I recommend a link to the `PotionUtils` method we all use here, its the one that reads a potion from a tag compound. A general link to potion utils is probably fine as well.